docs/roadmap.md

```diff
@@ -1,5 +1,24 @@
 # Roadmap
 
+## Feast 0.6 \(Feature Release\)
```

Comment:
Not sure where this came from, bad rebase?

Reply:
Apparently, I don't see any difference in my branch vs master on this file. So have no idea why it shows it here